src/app/donation-complete/donation-complete.component.html

```diff
@@ -151,11 +151,31 @@ <h2>Thank you!</h2>
         <p>Boost your impact by sharing on social media</p>
         <div class="social-icons-wrapper">
           <span class="separator"></span>
-          <biggive-social-icon service="Facebook" url="https://www.facebook.com/sharer/sharer.php?u={{ shareUrl }}" background-colour="tertiary" icon-colour="black" wide="true"></biggive-social-icon>
+          <biggive-social-icon
+            service="Facebook"
+            label-prefix="Share"
+            url="https://www.facebook.com/sharer/sharer.php?u={{ shareUrl }}" background-colour="tertiary"
```

Comment:
I know not a new issue, but maybe worth encodeURI'ing the shareUrl we use here now.